utilities/update_docs_version.sh

```diff
@@ -17,7 +17,11 @@ if [ "${RELEASED_VERSION##*-}" != "SNAPSHOT" ]; then
         sed -ri "s/:[0-9]+\.[0-9]+\.[0-9]+'/:${RELEASED_VERSION}'/g" ${item}/README.md
         sed -ri "s/\"[0-9]+\.[0-9]+\.[0-9]+\"/\"${RELEASED_VERSION}\"/g" ${item}/README.md
     done
-    
+
+    NEW_VERSION=${RELEASED_VERSION%.*}.$((${RELEASED_VERSION##*.}+1))-SNAPSHOT
+    echo "Changing version to $NEW_VERSION in gcloud-java-nio-example README"
+    sed -ri "s/gcloud-java-nio-examples-[0-9]+\.[0-9]+\.[0-9]+-SNAPSHOT/gcloud-java-nio-examples-$NEW_VERSION/g" gcloud-java-contrib/gcloud-java-nio-examples/README.md
```

Comment:
One thing I should add here is that there will be a small amount of time (during the release process) when the gcloud-java-nio-examples readme will be out of sync with the actual version of the project.  This script gets run while the project version is a non-snapshot.  Until the next PR goes through to update the version to x.x.x+1-SNAPSHOT, this readme will be out of date.  However, the amount of time that the readme will be out of sync should be small, since we typically push the version to x.x.x+1-SNAPSHOT as soon as the push to maven is complete (which usually takes under an hour).  An alternative approach to instruct the user to use a `find` command to get the proper version, and then use that environment variable to run the example. 

I think it's fine as is, but it's worth mentioning the alternative.